applications/FluidDynamicsApplication/fluid_dynamics_application.cpp

```diff
@@ -303,6 +311,7 @@ void KratosFluidDynamicsApplication::Register() {
     KRATOS_REGISTER_CONDITION("EmbeddedAusasNavierStokesWallCondition2D2N", mEmbeddedAusasNavierStokesWallCondition2D);
     KRATOS_REGISTER_CONDITION("EmbeddedAusasNavierStokesWallCondition3D3N", mEmbeddedAusasNavierStokesWallCondition3D);
     KRATOS_REGISTER_CONDITION("StokesWallCondition3D", mStokesWallCondition3D);
+    KRATOS_REGISTER_CONDITION("StokesWallCondition3D4N", mStokesWallCondition3D4N);
```

Comment:
Didn't we decide to use the Navier-Stokes condition?

Reply:
I see only two NS conditions `NavierStokesWallCondition2D2N` and `NavierStokesWallCondition3D3N`. So, how would I use the NS condition? Do you want me to add `NavierStokesWallCondition3D4N` also/instead?

Reply:
My point is that if we have the `NavierStokesWallCondition3D4N` we can get rid of the `StokesWallCondition`. 

Maybe we can do such clean up in a future PR. And just instantiate the Stokes one as you did. I have the feeling that this one is becoming way large. What do you think?

Reply:
I agree; it is worth taking the clean up in a separate PR.  Let us wrap this one up.